src/rgw/rgw_rest.h

```diff
@@ -391,6 +391,60 @@ class RGWInfo_ObjStore : public RGWInfo {
     ~RGWInfo_ObjStore() = default;
 };
 
+class RGWCreateRole_ObjStore : public RGWCreateRole {
+public:
+  RGWCreateRole_ObjStore() = default;
+  ~RGWCreateRole_ObjStore() = default;
+};
+
+class RGWDeleteRole_ObjStore : public RGWDeleteRole {
+public:
+  RGWDeleteRole_ObjStore() = default;
+  ~RGWDeleteRole_ObjStore() = default;
+};
+
+class RGWGetRole_ObjStore : public RGWGetRole {
+public:
+  RGWGetRole_ObjStore() = default;
+  ~RGWGetRole_ObjStore() = default;
+};
+
+class RGWModifyRole_ObjStore : public RGWModifyRole {
+public:
+  RGWModifyRole_ObjStore() = default;
+  ~RGWModifyRole_ObjStore() = default;
+};
+
+class RGWListRoles_ObjStore : public RGWListRoles {
+public:
+  RGWListRoles_ObjStore() = default;
+  ~RGWListRoles_ObjStore() = default;
+};
+
+class RGWPutRolePolicy_ObjStore : public RGWPutRolePolicy {
+public:
+  RGWPutRolePolicy_ObjStore() = default;
+  ~RGWPutRolePolicy_ObjStore() = default;
+};
+
+class RGWGetRolePolicy_ObjStore : public RGWGetRolePolicy {
+public:
+  RGWGetRolePolicy_ObjStore() = default;
+  ~RGWGetRolePolicy_ObjStore() = default;
+};
+
+class RGWListRolePolicies_ObjStore : public RGWListRolePolicies {
+public:
+  RGWListRolePolicies_ObjStore() = default;
+  ~RGWListRolePolicies_ObjStore() = default;
+};
+
+class RGWDeleteRolePolicy_ObjStore : public RGWDeleteRolePolicy {
+public:
+  RGWDeleteRolePolicy_ObjStore() = default;
+  ~RGWDeleteRolePolicy_ObjStore() = default;
+};
+
```

Comment:
since these ops are all specific to s3, maybe we don't need the usual `Op`, `Op_ObjStore`, `ObjStore_S3` hierarchy. can we just put them all in a separate `rgw_rest_role.cc`?